app/CurrikiGo/Moodle/Playlist.php

```diff
@@ -51,6 +51,18 @@ public function send(PlaylistModel $playlist, $data)
             "subject_name" => $subject_name ? : "None"
         ];
         $response = $this->client->request('GET', $web_service_url, ['query' => $rquest_params]);
+        if($response->getStatusCode() === 200 || $response->getStatusCode() === 201){
+            $lmsData = json_decode($response->getBody()->getContents());
+            $teacherInfo = new \stdClass();
+            $teacherInfo->user_id = null;
+            $teacherInfo->id = $lmsData[0]->data->id;
```

Comment:
please check $lmsData array first then add check for $lmsData[0] as well. Because in case of empty array it will break the code